contract_update_last_date_invoiced/models/contract_line.py

```diff
@@ -17,15 +17,17 @@ def action_update_last_date_invoiced(self):
         context.update(self.env.context)
         return {
             "type": "ir.actions.act_window",
-            "name": _("Update Last Date Invoiced"),
+            "name": _("Update Invoice Dates"),
             "res_model": "contract.update.last.date.invoiced",
             "view_type": "form",
             "view_mode": "form",
             "target": "new",
             "context": context,
         }
 
-    def _update_last_date_invoiced(self, last_date_invoiced, recurring_next_date):
+    def _update_contract_last_date_invoiced(
```

Comment:
We avoid changing method names because it breaks inheritance in dependent modules. Is there a good reason why you did it here?